analytics/views/activity_common.py

```diff
@@ -131,7 +132,8 @@ def realm_support_link(realm_str: str) -> Markup:
 
 
 def realm_url_link(realm_str: str) -> Markup:
-    url = get_realm(realm_str).uri
+    default_host = f"{realm_str}.{settings.EXTERNAL_HOST}"
```

Comment:
You want `host_for_subdomain` here to handle the `REALM_HOSTS` overrides.

Reply:
Right, thanks! Made that update.